test/integration/lib/foxboxHelper.js

```diff
@@ -12,9 +12,22 @@ var foxboxInstance;
 var helper = (function() {
 
   function removeUsersDB() {
-    var filePath = path.join(process.env.HOME,
-      '/.local/share/foxbox/users_db.sqlite');
-    fs.unlinkSync(filePath);
+    try {
```

Comment:
Nit: Duplication could be factorized in an internal function like `_removeFileIfItExists()`